option.go

```diff
@@ -69,6 +71,29 @@ func WithAttributeFilter(filter AttributeFilter) Option {
 	return &attributeFilterOption{filterAttribute: filter}
 }
 
+// WithLowerServerCardinality removes net.peer.port and net.peer.name attributes from server trace and span attributes.
+// This is recommended to use as net.peer server attributes have high cardinality.
+func WithLowerServerCardinality() Option {
+	return &withLowerServerCardinality{}
+}
```

Comment:
Two suggestions here:

1. Can we make this name more specific? Post-1.0, we won't be able to add more filtering to _this_ option without breaking back-compat. If/when we need to add more canned attr-filtering options, it'll be much easier if this option is named something specific like `WithoutHandlerPeerAttrs`.

2. Rather than defining a new struct, can this option return `WithAttributeFilter(someFunction)`?

Reply:
changed it to `WithoutServerPeerAttributes` because:
- WithoutServerNetPeer doesn't specify `Attributes`
- Server is the terminology of otel instead of Handler
- Attribute and Attributes are already established within this codebase instead of `Attrs`